### PR TITLE
Updating Docker healthcheck

### PR DESCRIPTION
**Pull request by nniehoff**
### Fixes: #1106

We need to figure out a health check for Celery Beat we can do that with this PR or another once since there is another issue #1102

### CODE REVIEW

**Comment by dgarros** [comment]
Would be good to fix this one in 1.1 too

**Comment by glennmatthews** [comment]
Can you confirm that the CPU/memory impact of the Celery worker healthcheck is reasonable using "nautobot server celery inspect ping..."? I'm seeing some remarkable CPU/memory spikes in a deployment with this command, I suspect perhaps https://github.com/celery/celery/issues/4079 is a factor here?

**Comment by nniehoff** [comment]
@glennmatthews it's still spikes but it is much much more performant than running `nautobot-server health_check`.  Having read that issue and some others if we have large jobs it's possible the ping won't return in time.  My hope is with #710 we can use something even better.  This is the CPU usage for the 2 celery pods running with demo.nautobot.com:


Screen Shot 2021-11-30 at 8 18 32 AM

File: development/docker-compose.build.yml [hunk 1]
```diff
@@ -8,6 +8,13 @@ services:
     image: "networktocode/nautobot-py${PYTHON_VER}:local"
     ports:
       - 8443:8443
+    healthcheck:
+
```
Comment (glennmatthews) [inline development/docker-compose.build.yml:12]: Remove empty line?

File: development/docker-compose.yml [hunk 2]
```diff
@@ -39,7 +49,10 @@ services:
       timeout: 5s
       start_period: 5s
       retries: 3
-      test: ["CMD", "nautobot-server", "health_check"]
+      test: 
+        - "bash"
+        - "-c"
```
Comment (glennmatthews) [inline development/docker-compose.yml:54]: For my education, why do we need `bash -c` here instead of `CMD` as in the cases above?

File: development/docker-compose.yml [hunk 3]
```diff
@@ -48,13 +61,13 @@ services:
     tty: true
   celery_beat:
     image: "networktocode/nautobot-dev-py${PYTHON_VER}:local"
-    entrypoint: "nautobot-server celery beat -l INFO -s /tmp/celerybeat-schedule"
```
Comment (glennmatthews) [inline development/docker-compose.yml:51]: IIRC the reason for `-s /tmp/celerybeat-schedule` was so that a `celerybeat-schedule` file wouldn't be created in the volume-mounted source directory. Is that no longer a concern, or what's the reason for this change?

**Comment by nniehoff** [comment]
> Would be good to fix this one in 1.1 too

While I don't think it's a big deal to backport this to 1.1 I would disagree and say people shouldn't be using the docker-compose files provided in this repo for production.  They should be using the docker-compose files from https://github.com/nautobot/nautobot-docker-compose and this fix should be applied there as well see https://github.com/nautobot/nautobot-docker-compose/issues/27

**Review by glennmatthews — APPROVED** [review]

**Review by jathanism — APPROVED** [review]
